x-pack/test/functional/apps/machine_learning/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -76,6 +77,11 @@ export default function({ getService }: FtrProviderContext) {
           await ml.dataFrameAnalyticsCreation.setJobId(testData.jobId);
         });
 
+        it('inputs the job description', async () => {
+          await ml.dataFrameAnalyticsCreation.assertJobDescriptionInputExists();
+          await ml.dataFrameAnalyticsCreation.setJobDescription(testData.jobDescription);
+        });
+
```

Comment:
We should also assert that the description is saved correctly during the test step `displays details for the created job in the analytics table` below. This requires a few additional changes in the table and the table service methods, but would be consistent with the way we check other job details.

Reply:
Thanks for taking a look! 😄 
Good call - added in 6116a586782001a074d9582df01434f18a3c8975